src/selectedtests/datasource/mongo_wrapper.py

```diff
@@ -56,3 +56,19 @@ def task_mappings(self):
         :return: task_mappings collection.
         """
         return self.client.selected_tests.task_mappings
+
+    def test_mappings_project_config(self):
```

Comment:
Would it make sense to have just a single project_config collection and keep track of both the test and task states in that collection? It seems like it could be useful to have all the details about a projects in a single location.

Reply:
Seconded